cmd/influx/cli/cli.go

```diff
@@ -994,7 +994,11 @@ func interfaceToString(v interface{}) string {
 	case float32, float64:
 		return fmt.Sprintf("%v", t)
 	default:
-		return fmt.Sprintf("%v", t)
+		if !escapeString {
+			return fmt.Sprintf("%v", t)
+		}
+		v := fmt.Sprintf("%q", t)
```

Comment:
Aside from binary data getting escaped, this will have an effect on 2 non-binary characters, `\` and `"`, which will get translated to `\\` and `\"` respectively.
The whole reason for doing this escaping on columnar output only was to only affect usage by humans, so this seems like a minor break. But open to other options (such as removing the escaping on `"`).